ReSharper.FSharp/src/FSharp/FSharp.Psi.Features/src/AICore/FSharpFileSummarizer.fs

```diff
@@ -78,16 +78,7 @@ type private FileSummarizerVisitor() =
     let addMember (memberDecl: IOverridableMemberDeclaration) (context: SummarizerContext) =
         let typeRepr = memberDecl.GetFcsSymbol() |> formatFcsSymbolType
         let accessorNames =
-            match memberDecl with
-            | :? IAccessorsNamesClauseOwner as prop ->
-                match prop.AccessorsClause with
-                | null -> Seq.empty
-                | clause -> clause.AccessorsNamesEnumerable |> Seq.map _.Name
-
-            | :? IMemberSignatureOrDeclaration as memberDecl ->
-                memberDecl.AccessorDeclarationsEnumerable |> Seq.map _.NameIdentifier.Name
-
-            | _ -> Seq.empty
+            memberDecl.AccessorDeclarations |> Seq.map _.NameIdentifier.Name
```

Comment:
I'd suggest having something like `SourceName`/`DeclaredName` on the accessor declaration, which would the case when the identifier is missing for some reason.